### ISSUE TITLE
auto-refund recent org members

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/action#2544]
AC
- [ ] if an org member is removed from an org and they were only there for <14 days, prorate their entire time.

Effort: 11 points

**Comment by mattkrick** [comment]
this applies exclusively to the remove from org mutation.
we'll need to track whether they have been removed or not, which means we'll need to create an `OrganizationUser` table. if there's time, we can refactor `orgUsers` and `userOrgs` to user the new table.
an org user has `orgId, userId, isActive, softUntil, isRefundedWhileSoft`. the last one tracks whether that user was added to the org & then removed. this will be important so folks can't just remove their entire team every 14 days when on a pro account.
when the email goes out, we set `softUntil` for 7 days
when removed, we check softUntil & isRefundedWhileSoft. if today < softUntil & they weren't refunded, then we refund them & tell them. if they were refunded, we provide client-side info telling them they were removed, but not refunded. if no longer soft, it's BAU.

we may want to include a "New" badge for all soft members. we should probably call them new instead of soft, anyways.
